news/2019-03-25-project-loved-week-of-march-24th.md

```diff
@@ -0,0 +1,225 @@
+
+[![](/wiki/shared/news/2019-03-25-project-loved-week-of-march-24th/mania/rhuzerv.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[Feryquitous feat. Sennzai - Rhuzerv](https://osu.ppy.sh/beatmapsets/846866#mania) by [Abraxos](https://osu.ppy.sh/users/5025064)  
+191 BPM, 3:34 | 4K 4.42★, 4K 4.50★
+
+If I had to describe Rhuzerv with one word, it would be "meticulous". Abraxos stands out as a charter for his combination of scrupulous attention to detail in a song and his unorthodox pattern use (e.g. difficult LN releases, generous use of explicit minijacks, more motion-focused patterning). You would be hard-pressed to find a charter who captures details in a song more unconventionally yet effective than Abraxos. There are two difficulties in this mapset, but Better Demons of Our Nature is just a cut of the other difficulty (Better Angels of Our Nature). I would recommend the latter difficulty. The chart itself is a heavily LN-focused technical hybrid: it has fast bursts, LNs of varying lengths and releases, grace notes and gallops, and a lot of minijacks. The chart is also accompanied with a relatively high OD for a LN chart: 8.2.
+
+As aforementioned, Rhuzerv stands out due to its meticulousness: Pitch relevance is tactfully used in this chart (pay attention to the synth in the intro and in certain later sections of the chart), minijacks are used to emphasise more salient sounds (a daring but effective accentuation), bursts and grace notes for harsher buzz noises, occasional repetition, chord allocation, and varying LN lengths based on the sounds that Abraxos is trying to capture. That list that I made isn't exhaustive—there are way many idiosyncrasies in this chart to write in a single write-up.
+
+The main point is this: the chart overall is very detailed and dynamic, and the details in the chart give the details of the track a life of their own. The chart also doesn't seem to border on excess in any capacity; a very tactful ear would be able to identify what a certain short LN, burst or type of double goes to without much problem. It most certainly helps that the chart is an unique and enjoyable experience for players as well. If you like LN release-oriented charts and technical charts, this chart is perfect for you.
+
+[![](/wiki/shared/news/2019-03-25-project-loved-week-of-march-24th/mania/back-to-the-gate.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[DJ Sharpnel - Back to the gate](https://osu.ppy.sh/beatmapsets/305124#mania) by [pporse](https://osu.ppy.sh/users/4991434) and [bbu2](https://osu.ppy.sh/users/4307930)  
+191 BPM, 5:19 | 7K 6.12★
+
+Primarily known for its appearance in the Semifinals in the osu!mania 7K World Cup 2016 and as a major stepping stone to chordstreaming and stamina practice, Back to the Gate is one of pporse's more well known and loved map overall.
```

Comment:
`Gate` -> `gate`?
```suggestion
Primarily known for its appearance in the Semifinals in the osu!mania 7K World Cup 2016 and as a major stepping stone to chordstreaming and stamina practice, Back to the gate is one of pporse's more well known and loved map overall.
```

Reply:
yeah I think the writer was working with old metadata